gamebook/option.py

```diff
@@ -1,20 +1,23 @@
 class Option(object):
-    def __init__(self, title, user_input, next_scene):
+    def __init__(self, title, user_input, next_scenes):
         self.title = title
         self.user_input = user_input
-        self.next_scene = next_scene
+        self.next_scenes = next_scenes
+
 
     def show_title(self): 
         return self.title
 
-    def show_next_scene(self):
-        next_scene_string = 'next scenes:'
-        for nextscene in self.next_scene:    
+
+    def show_next_scenes(self):
```

Comment:
show usually means you print things (know for next time) it doesn't really matter for this PR right now.